lpc55_sign/Cargo.toml

```diff
@@ -22,6 +22,7 @@ rsa = "0.6.1"
 lpc55_areas = { path = "../lpc55_areas", features = [] }
 serde = { version = "1", features = ["derive"] }
 toml = "0.5.6"
+parse_int = "0.6.0"
```

Comment:
Want to bump this in `lpc55_isp` and `lpc55_areas` as well, to avoid having multiple versions in the workspace?

Reply:
Done.